apps/chat/agent/schemas.py

```diff
@@ -17,11 +17,12 @@ class WeekdaysEnum(int, Enum):
 
 
 class RecurringReminderSchema(BaseModel):
-    datetime_due: datetime = Field(description="the first (or only) reminder due date in ISO 8601 format")
+    datetime_due: datetime | None = Field(description="the first (or only) reminder start date in ISO 8601 format")
```

Comment:
Thanks for catching this. This one should not have `| None`, but to answer your question: Nothing, since this is not the way to tell the bot that this could be `None`. We must use `Field(default=None)` if we want the bot to know about this possibility